src/docs/patterns/appbar.mdx

```diff
@@ -1,43 +1,86 @@
 
-In some situations, you may want to present more information in an app bar, such as a large graphic, buttons, or multiple lines of text. In these cases, you can use a taller app bar that will shrink as the user scrolls down the page and be replaced by a basic app bar. This allows you the flexibility to present an eye-catching banner without permanently reducing the amount of usable space on the page. This is a pattern most commonly used for mobile applications, but could also be used at a larger scale.
+In some situations, you may want to present more than just a title and subtitle in an app bar, such as a large graphic, buttons, or multiple lines of text. In these cases, you can use a taller app bar that will shrink as the user scrolls down the page until it becomes a basic app bar. This allows you the flexibility to present an eye-catching banner without permanently reducing the amount of usable space on the page. This is a pattern most commonly used for mobile applications, but could also be used for web.
 
-## Search Bar
+The collapsible appbar should transition smoothly between the expanded and collapsed states — this means you will need to account for how the app bar will look throughout the transition. Because elements will disappear as the app bar shrinks, you'll need to use dynamic text scaling, opacity transitions, and / or other techniques to do this smoothly, depending on your content.
+
+<Divider />
+
+# Search Bar
 
 <ImageGrid
     images={[Image3, Image4]}
     caption={'When the search icon is clicked, a white search bar will slide in and cover the default tool bar. '}
 />
 <DemoCard repository={'search-bar'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Fapp-bar%2Fsearch-bar%2Fsearch-bar.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/search-bar/index.tsx&initialpath=search'} ionic reactNative />
 
-A very common pattern seen in many types of applications is the ability to search for data on a page. This is achieved by placing a search icon/button in the app bar. When this button is clicked, a white Search Bar slides in from the right to cover the default toolbar. As the user enters text into the search field, the data is filtered in real-time.
+A very common pattern seen in many types of applications is the ability to search for data on a page. This is achieved by placing a search icon/button in the app bar. When this button is clicked, a white search bar slides in from the right to cover the default app bar. As the user enters text into the search field, the data is filtered in real-time. A back icon will close the Search Bar and clear the search results.
```

Comment:
I think not all the applications do a mega white search bar like our docit. A lot of them do it in a small text field, like what we have on the iconography page icon browser. 

![image](https://user-images.githubusercontent.com/8997218/92785082-207ce280-f375-11ea-9c77-75f57ac9d8ef.png)

Reply:
Do we want to include the mini one as another option? What are the rules for using one versus the other?

Reply:
I am more inclined to yes, but I’d say have some design meetings around it.